interface-definitions/include/firewall/address.xml.i

```diff
@@ -0,0 +1,36 @@
+<!-- include start from firewall/address.xml.i -->
+<leafNode name="address">
+  <properties>
+    <help>IP address, subnet, or range</help>
+    <valueHelp>
+      <format>ipv4</format>
+      <description>IP address to match</description>
+    </valueHelp>
+    <valueHelp>
+      <format>ipv4net</format>
+      <description>Subnet to match</description>
+    </valueHelp>
+    <valueHelp>
+      <format>ipv4range</format>
+      <description>IP range to match</description>
+    </valueHelp>
+    <valueHelp>
```

Comment:
Please see https://github.com/vyos/vyos-1x/blob/current/interface-definitions/include/nat-address.xml.i and use the same syntax on the format identifier.

Reply:
done

Reply:
But for ipv6 we don't have such validators yet
```
      <validator name="ipv6-address-exclude"/>
      <validator name="ipv6-prefix-exclude"/>
      <validator name="ipv6-range-exclude"/>
```